libvirt/tests/src/virsh_cmd/domain/virsh_destroy.py

```diff
@@ -74,6 +100,49 @@ def run(test, params, env):
                                unprivileged_user=unprivileged_user,
                                uri=uri, debug=True).exit_status
         output = ""
+
+        if start_destroy_times:
+            status = 0
+            try:
+                # RHEL9 will config virtqemud.service
+                modify_service = "virtqemud.service"
+                if distro.detect().name == 'rhel' and \
+                        int(distro.detect().version) < 9:
+                    modify_service = "libvirtd.service"
```

Comment:
We should get configuration file through libvirt daemons mode, but not distribution version  as I know. So I think it's better to update it to
```suggestion
                modify_service = utils_libvirtd.Libvirtd('virtqemud').service_name
```
OR 
```suggestion
                modify_service = "virtqemud.service"
                if utils_split_daemons.is_modular_daemon():
                      modify_service = "libvirtd.service"
```
What do you think?